docs/overview/index.md

```diff
@@ -7,4 +7,5 @@ tasks
 simulators
 overview_computational_resources
 infrastructure
+uninstall_inductiva
```

Comment:
This should be in user_reference (the last section) because it is not crucial to the initial steps of the user.